kbase-extension/static/kbase/js/widgets/appWidgets2/input/unused/dataAttachmentInput.js

```diff
@@ -448,7 +448,7 @@ define([
             const events = Events.make(),
                 control = makeInputControl(events, bus);
 
-            dom.setContent('input-container', control);
+            ui.setContent('input-container', control);
```

Comment:
All the functions in `dom.js` are also in `ui.js`, so we can just swap out all the dom.js mentions.